test/GADM.jl

```diff
@@ -36,22 +36,38 @@ end
     @test_throws ArgumentError GADM.get("ind")    
     @test_throws ArgumentError GADM.get("")   
     @test_throws ArgumentError GADM.get("IND4")   
-    # valid country codes, children=false
-    parent = GADM.get("IND")
-    @test parent isa NamedTuple
-    @test GeoInterface.geomtrait(parent.geom[1]) isa MultiPolygonTrait
-    # valid country code, children=true
-    parent, children = GADM.get("IND";children=true)
-    @test parent isa NamedTuple
-    @test children isa NamedTuple
-    @test GeoInterface.geomtrait(parent.geom[1]) isa MultiPolygonTrait
-    @test length(children) == 11 #number of fields in named tuple
-    geometries = Tables.getcolumn(children, Symbol("geom"))
+    # valid country code
+    country = GADM.get("IND")
+    @test country isa NamedTuple
```

Comment:
I think we want to test that the output satisfies the Tables.jl interface, that is Tables.istable(country). Can you please double check all tests that check against NamedTuple and replace by Tables.istable?

Reply:
ok